### DISCUSSION TITLE
Add Macos ARM to prebuilt release binaries

### DISCUSSION THREAD

**Discussion by pnegahdar** [meilisearch/product#518] [Feedback & Feature Proposal]
Can we add a macos-arm to the prebuilt binaries in the [releases](https://github.com/meilisearch/meilisearch/releases)? M1/2 macs are really popular now  in the dev world.

**Comment by curquiza** [comment]
Hello @pnegahdar 

You can download the `meilisearch-macos-amd64` binary, it will work thanks to rosetta. Every Meili developer has a M1 and the binary works on their machine.
Sorry if I misunderstood your need, let me know!

**Reply by pnegahdar** [reply]
:) thanks for the reply. You are correct, and that is how I am using it right now. 

It'd be great to run native (performance penalty from rosetta, indexing is very cpu intensive). And I have the meilisearch binary embedded in an electron application so having a stable URL to grab the latest native binary would be fantastic. 

By the way Homebrew seems to build and publish the arm bottles: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/meilisearch.rb#L14, I'm looking into how to access those from the registry here: https://github.com/Homebrew/homebrew-core/pkgs/container/core%2Fmeilisearch

**Reply by curquiza** [reply]
I discussed with the dev team, this is something we could do in the future, with other new binaries like other Windows! 😄
Not in v0.29.0 we will release soon, but more in v0.30.0. We also want to change the binary names in the same time, and we have to ensure it's not too much breaking for the teams working with us first!
Thanks for the suggestion!

**Comment by gmourier** [comment]
Hey everyone 👋

I'm locking that discussion since it has been solved.

Feel free to reopen a discussion if it's needed 🙇‍♂️

Thank you!
